pom.xml

```diff
@@ -354,6 +356,31 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>owasp-check</id>
+            <activation>
+                <property>
+                    <name>owasp-check</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.owasp</groupId>
+                        <artifactId>dependency-check-maven</artifactId>
+                        <version>${owasp-dependency-check-plugin.version}</version>
+                        <configuration>
+                            <!-- Disable Net Analyzer -->
+                            <assemblyAnalyzerEnabled>false</assemblyAnalyzerEnabled>
+                            <nugetconfAnalyzerEnabled>false</nugetconfAnalyzerEnabled>
+                            <nuspecAnalyzerEnabled>false</nuspecAnalyzerEnabled>
+                            <retireJsAnalyzerEnabled>false</retireJsAnalyzerEnabled>
+                        </configuration>
+                    </plugin>
+                </plugins>
+                <defaultGoal>org.owasp:dependency-check-maven:check</defaultGoal>
```

Comment:
Not sure why but here a non qualified `dependency-check:check` does not work, but from the user's perspective, if they want to run from the command line it is still `mvn -Dwasp-check`, now not only in the extension directory but in higher-level dirs up to the root